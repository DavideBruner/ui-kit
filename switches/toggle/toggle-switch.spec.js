import React from 'react';
import { shallow } from 'enzyme';
import ToggleSwitch from './toggle-switch';

const createTestProps = custom => ({
  isDisabled: false,
  isChecked: false,
  isMouseOver: false,
  ...custom,
});

describe('<ToggleSwitch>', () => {
  describe('rendering', () => {
    let props;
    let wrapper;

    it('outputs correct tree', () => {
      props = createTestProps();
      wrapper = shallow(<ToggleSwitch {...props} />);

      expect(wrapper).toMatchSnapshot();
    });
  });
});
