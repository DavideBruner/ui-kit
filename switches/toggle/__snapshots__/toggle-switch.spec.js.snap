// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToggleSwitch> rendering outputs correct tree 1`] = `
<div
  className="toggle-container-big"
>
  <ToggleButton
    isChecked={false}
    isDisabled={false}
    isMouseOver={false}
    size="big"
  />
  <ToggleBar
    isChecked={false}
    isDisabled={false}
    isMouseOver={false}
    size="big"
  />
</div>
`;
